mautic/mautic
# [UI] Accessibility features in user profile

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

It adds accessibility features:
- Select whether transitions remain smooth or go instant
- Toggle strong borders around clickable elements and dashed outline style
- Adjust the transparency and blur effects to improve readability or visual clarity
- Toggle the visibility of underlines on links that are adjacent to text

![image](https://github.com/user-attachments/assets/d8674f2e-52f8-42cb-865e-067da8f47810)




https://github.com/user-attachments/assets/2e2ea783-bbbd-4ff2-9a6c-f7d49de8d237









---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Click on the user menu in the header
3. Click to open your profile "Account" 
4. Open the tab Accessibility 

**Testing Smooth transitions**
5. Hover your mouse over the **Save** and **Cancel** buttons to memorize the smooth transition
6. Now set _Smooth transitions_ to **Disabled**
7. Hover again over those buttons, you'll notice their color changes instantly

**Testing borders and outline**
8. Click on the tab Accessibility and memorize how the link looks
9. Set Borders and outline style to **Enabled**
10. Click again: you'll see a dashed border appearing around the text to help you locate the focus points on the screen
11. Open the Account details tab
12. Check how all fields now have a strong border on all sides

**Testing reduced transparency and blur**
13. Go to Contacts
14. Open the actions dropdown menu
15. Select **Export to CSV**
16. See the flash message appearing with blurred background
17. Now go back to your account, in the Accessibility tab
18. Set Reduce transparency and blur to **Reduced**
19. Repeat the export to see

**Testing link underlines**
20. On the Accessibility tab, set Link underlines to **Enabled**
21. Check the links on tables, alerts, accordions, paragraphs and also the links to each tab
22. They all have underlines
